@xen-orchestra/web-core/lib/components/UiLegend.vue

```diff
@@ -0,0 +1,97 @@
+<!-- v1.0 -->
+<template>
+  <li class="legend">
```

Comment:
- Component class should follow the component name
- Variant class should be added on root element.

```suggestion
  <li :class="color" class="ui-legend">
```